bokeh/core/properties.py

```diff
@@ -1231,7 +1243,7 @@ def from_json(self, json, models=None):
                 attrs = {}
 
                 for name, value in iteritems(json):
-                    prop = self.instance_type.lookup(name)
+                    prop = self.instance_type.lookup(name).descriptor
```

Comment:
@bryevdv, yes. This line appears in the traceback in the issue. The remainder of changes is to allow `assertEquals()` to work.